x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -324,6 +363,12 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().onOrAfter(Version.V_7_10_0)) { // TODO: Remove after backport
             out.writeString(resultPosition);
         }
+        if (out.getVersion().onOrAfter(Version.V_7_12_0)) {
```

Comment:
Shouldn't this be `V_8_0_0` for now and once backported switch to `V_8_0_0`?

Reply:
@matriv @bpintea Actually, the new test I added - https://github.com/elastic/elasticsearch/pull/68962/files#diff-7e1220049ee22470c58a915155e9d9bd3ef7742523a6bfd0e71b8c463df3e638 - won't let me use V_8_0_0 because a version 8.0 of ES should be compatible with the latest version 7.x of ES (which is 7.12). And the test in `AbstractBWCSerializationTestCase` is doing serialization and deserialization on 7.12 and compares the deserialized instance with the original (8.0) instance. And the test fails, because the two instances should be equal.